### PR TITLE
Report Model

### LINKED ISSUE #25: Report Model: Mortgage equation calculators
- [x] SOLVES FOR P: Add model function for calculating principal from rent
- [x] SOLVES FOR M: Add model function for calculating monthly principal from `principal` from the report POST above

### PR DESCRIPTION
**Pull request by Arique1104**
This PR...
## Changes
- Adds Report Model
- Adds documentation regarding the request and response cycle of report and education

## Checklist 
- [x] passes all tests on local machine
- [x] added new dependencies (include set-up or new gems)
- [ ] updated documentation / read-me
- [ ] added a test
## Linked tickest / issues 
Fixes #
Closes #25

### CODE REVIEW

File: api/models/report.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+class Report(db.Document):
+    salary = db.FloatField(required=True)
+    zipcode = db.IntField(required=True)
+    credit = db.IntField(required=True)
+    monthly_debt = db.FloatField(required=True)
+    downpayment_savings = db.FloatField(required=True)
+    downpayment_percentage = db.IntField(required=True)
+    rent = db.FloatField()
+    goal_principal = db.FloatField()
```
Comment (BlakeDonn) [inline api/models/report.py:11]: Everything looks to be in order here!

File: mvc.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
+# Psuedo Code
+
+
+## Model
+- Update the model fields to match the front end requirements.
+- Does education have any functions?  No, only for ReportModel. those functions talk directly to database AND/OR helpers
+
+doesn't have a user field...once they sign in
+
```
Comment (BlakeDonn) [inline mvc.md:9]: Thanks for the notes here, helps to visualize the dev process

File: README.md [hunk 3]
```diff
@@ -9,6 +9,196 @@
...
+    }
+  }
+}
+```
+
+### Education Model
+#### REQUEST
+```
+User clicks submit on survey
+# GET http://localhost:5000/api/v1/education
```
Comment (BlakeDonn) [inline README.md:107]: I think we will actually be making this request on app load, or somewhere higher up in the process. 

I think "User clicks submit on survey" could be replaced with "user loads the application"

**Review by BlakeDonn — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Arique1104) [reply README.md:107]: That's cool!  Does that help with load time?

**Review by EHale64 — APPROVED** [review]

File: api/models/report.py (hunk 1 above)
Comment (priyapower) [reply api/models/report.py:11]: Yay!

File: mvc.md (hunk 2 above)
Comment (priyapower) [reply mvc.md:9]: Awesome, in the next PR we will absolutely release better notes, especially the data structure. Some updates occurred as we began coding and realized we needed another Model

**Review by priyapower — APPROVED** [review]

**Pull request merged**